### DISCUSSION TITLE
Reduced Dynamic Orbits - satellite model / material coefficents

### DISCUSSION THREAD

**Discussion by Bemtgenj** [groops-devs/groops#184] [Q&A]
Hello,

We want to evaluate the impact on the material coefficients (absorption, emission - infrared/visible...) in the satellite model to see their impact on the Cubesat reduced dynamic orbits. In addition to running a "no satellite model" another step would be to run the script with surface areas and dimensions only (no material coefficients). When i look at the script for the satellite model creation the material coefficients parameter are required in the script and i think that adding a value of 0 to those parameters does not mean that they are not used in the model but means that they are set to 0. Is there an option in the script to only use the areas? Or is the only available solution to run a full satellite model but disable the i.e solar radiation, albedo,... parameters. 

TXS for the help and feedback

Cheers
Jean

**Comment by tmayerguerr** [comment]
Hi Jean,

I am not clear what you mean by a model without material properties.

The material properties are needed to calculate the radiation pressure. The coefficients indicate how much of the incoming (visible) solar radiation is reflected, absorbed or diffusely reflected. They should therefore add up to 1 (100%). 
The same applies to infrared radiation from the Earth. If you set all coefficients to 0, the respective effects will not be calculated (or to be precise: multiplied with 0).

regards
Torsten

**Reply by Bemtgenj** [reply]
Hi Torsten,

thanks very much for the feedback.
What i meant was that we just have the CubeSat Surfaces in the satellite model but not the material properties.. It is very difficult to find the coefficients for the different material, so I thought we would run a model with just the surfaces, as i did not want to have "wrong" estimates for material coefficients. As a start i had used GRACE FO material coefficients for example for the solar panels,... I also thought about putting the values at 0 but then i thought that is still a values and so it is not a version were i ignoe them.

I hope this make it a bit clearer. At the end the issue is finding for every material (solar panels, side walls,....) the correct coefficients and if not available the next best estimate

Txs again got the help and the feedback.

Cheers
Jean
